oauth2-authorization-server/src/test/java/org/springframework/security/oauth2/server/authorization/client/TestRegisteredClients.java

```diff
@@ -39,6 +39,18 @@ public static RegisteredClient.Builder registeredClient() {
 				.scope("scope1");
 	}
 
+	public static RegisteredClient.Builder registeredClientEncodedUri() {
```

Comment:
Remove this and leverage the existing `TestRegisteredClients.registeredClient()` and add the url-encoded redirect-uri